openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -241,11 +241,11 @@ jQuery(function () {
             .then(module => module.init(readingLogConfig));
     }
 
-    const pageEl = $('#page-barcodescanner');
-    if (pageEl.length) {
-        import(/* webpackChunkName: "page-barcodescanner" */ './page-barcodescanner')
-            .then((module) => module.init());
-    }
+    // const pageEl = $('#page-barcodescanner');
+    // if (pageEl.length) {
+    //     import(/* webpackChunkName: "page-barcodescanner" */ './page-barcodescanner')
+    //         .then((module) => module.init());
+    // }
```

Comment:
This is looking great, let's delete this and the other JS/CSS files about `page-barcodescanner`! 😊